app/views/steps/conviction/compensation_paid_amount/edit.html.erb

```diff
@@ -1,23 +1,15 @@
 <% title t('.page_title') %>
+<%= step_header %>
 
-<div class="govuk-width-container">
-  <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
+    <%= step_subsection %>
```

Comment:
I think for now it is ok to continue using this helper, until we've finished all the steps. Then we can go back and change this to use the native `caption` that the gem offers.

Reply:
Updated on https://github.com/ministryofjustice/disclosure-checker/pull/377/commits/3e9fc6b34d11c19de05b11512015dd310228b2d3

Reply:
Got a story for this one: https://mojdigital.teamwork.com/#/tasks/21724564